specification/widget/resource-manager/Microsoft.Widget/WidgetManagement/tspconfig.yaml

```diff
@@ -0,0 +1,49 @@
+parameters:
+  "service-dir":
+    default: "sdk/widgetmanagement"
+emit:
+  - "@azure-tools/typespec-autorest"
+options:
+  "@azure-tools/typespec-autorest":
+    use-read-only-status-schema: true
+    emitter-output-dir: "{project-root}"
+    # TODO: Does anything need this set, if it's not used in output-file?  Currently required by TSV.
+    azure-resource-provider-folder: "resource-manager"
+    output-file: "{version-status}/{version}/widgetmanagement.json"
+    arm-types-dir: "{project-root}/../../../../common-types/resource-management"
+  "@azure-tools/typespec-csharp":
+    flavor: azure
+    package-dir: "Azure.ResourceManager.Widget"
+    clear-output-folder: true
+    model-namespace: true
+    namespace: "{package-dir}"
+  "@azure-tools/typespec-python":
+    package-dir: "azure-mgmt-widget"
```

Comment:
Can you clarify for some of these configurations that don't have an explicit 'service' config (e.g., "@azure-tools/typespec-python", "@azure-tools/typespec-csharp", "@azure-tools/typespec-ts", etc.) is the expectation that these are per namespace? Trying to understand what this would like if we have multiple services in an RP (in this case widget is the name of RP and service within it, so it's not clear to me how `widget` is being referenced in these sections.)

An example where this distinction for service within RP is clear is :
"@azure-tools/typespec-java":
    package-dir: "azure-resourcemanager-widget"
    namespace: "com.azure.resourcemanager.widget"
    service-name: "Widget" # human-readable service name, whitespace allowed
    flavor: azure

Reply:
Would it help if we added multiple services and/or RP namespaces to the sample?  If so, I can coordinate this.

Reply:
Yes, I think having multiple services in the RP as part of the sample would be helpful. 
As for SDK settings, understood that is not part of this test and might be outside of this scope, but generally speaking at some point, it would be beneficial to have the latest samples reflect these accurately. 
For context, as a new RP team, we had to recently figure this for our RP which has multiple services, and it was not very apparent to us how these should be configured for this scenario.